doc/dev/ROADMAPS_AND_RELEASES.md

```diff
@@ -0,0 +1,29 @@
+# Roadmaps and release cycles
```

Comment:
Don't you need to add the file somewhere to make it available in f3d.app?

Reply:
Right I forgot, added.